staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4264,6 +4264,36 @@ type PodResourceClaimStatus struct {
 	ResourceClaimName *string `json:"resourceClaimName,omitempty" protobuf:"bytes,2,opt,name=resourceClaimName"`
 }
 
+// PodExtendedResourceClaimStatus is stored in the PodStatus for each extended
+// resource requests backed by DRA. It stores the generated name for
+// the corresponding special ResourceClaim created by scheduler.
+type PodExtendedResourceClaimStatus struct {
+	// RequestMapping identifies the mapping of <container, extended resource backed by DRA> to  device request.
+	// +patchMergeKey=requestName
+	// +patchStrategy=merge,retainKeys
+	// +listType=map
+	// +listMapKey=requestName
+	// +featureGate=DynamicResourceAllocation
```

Comment:
```suggestion
```

The entire struct is covered by `DRAExtendedResource`.

Reply:
removed.